webapp/home/subdomains/genome/content/annotation.py

```diff
@@ -131,20 +131,71 @@
 
 workflows = [
     {
-        "title_html": 'TBC',
-        "description_html": """<p>TBC</p>""",
+        "title_html": 'Annotation with Maker',
+        "description_html": """
+            <p>
+              Annotates a genome using multiple rounds of Maker, including gene prediction using SNAP and Augustus
+              <br>
+              <br>
+              Tools:
+              <code>maker</code>
+              <code>snap</code>
+              <code>augustus</code>
+              <code>busco</code>
+              <code>jbrowse</code>
+            </p>""", 
         "inputs": [
+             {
+                'datatypes': ['fasta'],
+                'label': 'genome',
```

Comment:
`label` should most likely be capitalised - and may I suggest "Genome assembly" for consistency with other items?